src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
```

Comment:
fix it
![image](https://user-images.githubusercontent.com/98764641/232715864-e6ea963f-f11c-4be8-912b-a21beba57acd.png)
if you have questions, feel free to ask in the chat